.github/workflows/ui_tests.yml

```diff
@@ -43,6 +43,7 @@ jobs:
       - name: Run Playwright tests
         env:
           NEXT_PUBLIC_PROJECT_ID: ${{ secrets.NEXT_PUBLIC_PROJECT_ID }}
+          NEXTAUTH_SECRET: ${{ secrets.TESTS_NEXTAUTH_SECRET }}
```

Comment:
Does this need to be a securely random secret value or would something arbitrary like "password" suffice? Since this server is only accessible inside the runner and it's not protecting any private info

Reply:
as far as I understand it can be any random value. However, I don't see a problem with this approach. Dislike putting anything that's suffixed with `SECRET` in cleartext or a random value in code